docs/advanced/api/rest.md

```diff
@@ -2,161 +2,189 @@
+    + asset: "NODIVISIBLE" (required, string) - Name of the Asset
+
+### View an Asset Detail [GET]
+
++ Response 200 (application/json)
+
+        {
+            "asset": "NODIVISIBLE",
+            "asset_longname": None,
+            "description": "No divisible asset",
+            "divisible": False,
+            "issuer": "mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc",
+            "locked": False,
+            "owner": "mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc",
+            "supply": 1000
+        }
+
+### View an Asset Balances [/assets/{asset}/balances]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "address": "1_mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc_mtQheFaSfWELRB2MyMBaiWjdDm6ux9Ezns_2",
+                "asset": "XCP",
+                "quantity": 300000000
+            },
+            {
+                "address": "2MyJHMUenMWonC35Yi6PHC7i2tkS7PuomCy",
+                "asset": "XCP",
+                "quantity": 46449548498
+            },
+            ...
+        ]
+
+### View an Asset Orders [/assets/{asset}/orders]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "tx_index": 11,
+                "tx_hash": "1899b2e6ec36ba4bc9d035e6640b0a62b08c3a147c77c89183a77d9ed9081b3a",
+                "block_index": 310010,
+                "source": "mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc",
+                "give_asset": "XCP",
+                "give_quantity": 100000000,
+                "give_remaining": 100000000,
+                "get_asset": "BTC",
+                "get_quantity": 1000000,
+                "get_remaining": 1000000,
+                "expiration": 2000,
+                "expire_index": 312010,
+                "fee_required": 900000,
+                "fee_required_remaining": 900000,
+                "fee_provided": 6800,
+                "fee_provided_remaining": 6800,
+                "status": "open"
+            },
+            ...
+        ]
+
+# Groupe Address
+
+Resources related to address in the API.
+
+### View an Address Balances [/assets/{address}/balances]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "address": "2MyJHMUenMWonC35Yi6PHC7i2tkS7PuomCy",
+                "asset": "MYASSET",
+                "quantity": 300000000
+            },
+            {
+                "address": "2MyJHMUenMWonC35Yi6PHC7i2tkS7PuomCy",
+                "asset": "XCP",
+                "quantity": 46449548498
+            },
+            ...
+        ]
+
+## Order [/orders/{tx_hash}]
+
+An Order object has the following attributes:
+
++ tx_index
++ tx_hash
++ block_index
++ source
++ give_asset
++ give_quantity
++ give_remaining
++ get_asset
++ get_quantity
++ get_remaining
++ expiration
++ expire_index
++ fee_required
++ fee_required_remaining
++ fee_provided
++ fee_provided_remaining
++ status
+
++ Parameters
+    + tx_hash: "1899b2e6ec36ba4bc9d035e6640b0a62b08c3a147c77c89183a77d9ed9081b3a" (required, string) - tx_hash of the order
```

Comment:
"`tx_hash` of the order" should maybe be "hash of the transaction that created the order" to be more understandable

Reply:
done!